tests/entrypoints/test_openai_server.py

```diff
@@ -119,7 +119,7 @@ def zephyr_lora_files():
     return snapshot_download(repo_id=LORA_NAME)
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
```

Comment:
hmm but this means we will be creating the test server for each step which is pretty bad latency hit. can you change this to get or create the actor? (it currently pass because ray is cleaned up as well) but we do want one shared server for all the tests. (the client fixture below is fine)

Reply:
Removed Ray shutdown from the global conftest. I will assume if tests use ray it will handle shutdown on its own (which is same as status quo).